.github/workflows/ci-e2e.yml

```diff
@@ -246,7 +246,11 @@ jobs:
               uses: actions/setup-node@v3
               with:
                   node-version: 18
+                  #todo understand this line
                   cache: pnpm
+                  cache-dependency-path: |
+                      **/package-lock.json
+                      /home/runner/.cache/Cypress
```

Comment:
the way i understand `cache-dependency-path` is that it uses these files as input for checking when to generate a new cache. what we want to do instead is to add `/home/runner/.cache/Cypress` to the cached dependencies.